.github/workflows/cd.yaml

```diff
@@ -0,0 +1,138 @@
+name: Build and Deploy AO/HyperBEAM
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  GCP_IMAGE_NAME: hyperbeam-image
+  GCP_PROJECT: hyperbeam-cd
```

Comment:
This project was made specifically for this branch. It is assumed that the GCP project with this name already exists, hence if a new project needs to be used, then that needs to be created prior to changing this line.